src/Mjolnir.ts

```diff
@@ -562,7 +452,7 @@ export class Mjolnir {
 
         // Check for updated ban lists before checking protected rooms - the ban lists might be protected
         // themselves.
-        const policyList = this.policyLists.find(list => list.roomId === roomId);
+        const policyList = this.policyListManager.lists.find(list => list.roomId === roomId);
```

Comment:
The direct access to the lists seem wrong, at the least we should ensure `.lists` returns a shallow copy. But we should think about avoiding access all together.

Also see https://github.com/Gnuxie/Draupnir/pull/5/files#r1086994746 which has a similar access above